resources/js/app.js

```diff
@@ -16,7 +17,31 @@ if (fakeID) {
 	serverURL.searchParams.set("fake-id", fakeID);
 }
 
+String.prototype.format = function () {
+	var formatted = this;
+	for (var i = 0; i < arguments.length; i++) {
+		var regexp = new RegExp("\\{" + i + "\\}", "gi");
+		formatted = formatted.replace(regexp, arguments[i]);
```

Comment:
Is order preserved when iterating through a map in JS?  Due to the way string replacement happens we may want to ensure the replacements happen in a particular order. e.g. We might want to replace "English Channel" before we replace "English".

Reply:
The arguments is an array, right? So it should just replace `{0}` with the first argument, etc.